contracts/emissions/ForwarderRecipient.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity 0.8.6;
+
+// Internal
+import { IRewardsDistributionRecipient } from "../interfaces/IRewardsDistributionRecipient.sol";
+import { IRootChainManager } from "../interfaces/IRootChainManager.sol";
+import { InitializableRewardsDistributionRecipient } from "../rewards/InitializableRewardsDistributionRecipient.sol";
+import { Initializable } from "../shared/@openzeppelin-2.5/Initializable.sol";
+
+// Libs
+import { IERC20, SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+/**
+ * @title  ForwarderRecipient
+ * @author mStable
+ * @notice transfers any received reward tokens to another contract or account.
+ * @dev     VERSION: 1.0
+ *          DATE:    2021-10-28
+ */
+contract ForwarderRecipient is
```

Comment:
My vote is that this `ForwarderRecipient` is:
a) Immutable (i.e. not a proxy)
b) `Ownable` by whoever is in charge of the recipient - allowing them to call `setEndRecipient`
c) `onlyGovernor` can still be in charge of rewardsDistributor